aa_stripe/serializers.py

```diff
@@ -17,14 +18,43 @@ class Meta:
         fields = ["stripe_card_id", "last4", "exp_month", "exp_year"]
 
 
+class StripeCardUpdateSerializer(ModelSerializer):
+    stripe_token = JSONField()
+    should_be_default = BooleanField()
```

Comment:
required=False, otherwise it will only accept True, besides, I'd prefer if it was called "set_default"